tests/natural_language/LanguageToolBearTest.py

```diff
@@ -9,10 +9,11 @@
 try:
     import language_check
     import guess_language
+    language_check
```

Comment:
this approach adds two lines, and works with pyflakes natively.
your solution adds or amends two lines, and only works within coala.
I'm not seeing the benefit, especially not work holding up this patch and potentially conflicting with other patches incoming.  If necessary we can switch later.